content/version/2/0/code_of_conduct.nl.md

```diff
@@ -0,0 +1,93 @@
+
+## Toepassingsgebied
+
+Deze gedragscode is van toepassing binnen alle gemeenschappelijke ruimtes, en is ook van toepassing wanneer een individu officieel de gemeenschap in openbare ruimtes vertegenwoordigt.
+Voorbeelden van het vertegenwoordigen van onze gemeenschap zijn onder meer het gebruik van een officieel e-mailadres, posten via een officieel social media-account, of optreden als een aangestelde vertegenwoordiger bij een online of offline evenement.
+
+## Handhaving
+
+Beledigend, intimiderend of anderszins onaanvaardbaar gedrag kan gerapporteerd worden aan de gemeenschapsleiders die verantwoordelijk zijn voor handhaving via [CONTACTMETHODE INVOEGEN].
+Alle klachten zullen onmiddellijk en eerlijk worden beoordeeld en onderzocht.
+
+Alle gemeenschapsleiders zijn verplicht om de privacy en veiligheid van de verslaggever van een incident te respecteren.
+
+## Handhavingsrichtlijnen
+
+Gemeenschapsleiders zullen onderstaande "Community Impact Guidelines" volgen bij het bepalen van de consequenties van elke handeling die zij in strijd met de Gedragscode achten:
+
+### 1. Correctie
+
+**Impact op de gemeenschap**: gebruik van ongepast taalgebruik of ander gedrag dat onprofessioneel wordt geacht, of niet welkom in de gemeenschap.
+
+**Gevolg**: een persoonlijke, schriftelijke waarschuwing van community leiders, verduidelijking over de aard van de overtreding en een uitleg waarom het gedrag ongepast wordt geacht. Mogelijk wordt om een ​​openbare verontschuldiging gevraagd.
+
+### 2. Waarschuwing
+
+**Impact op de gemeenschap**: een overtreding door een enkel incident of reeks van acties.
+
+**Consequentie**: een waarschuwing met gevolgen als het gedrag voortduurt. Geen interactie met de betrokken personen, inclusief ongevraagde interactie met degenen die de gedragscode handhaven, voor een bepaalde periode. Dit omvat het vermijden van interacties in gemeenschappelijke ruimtes en externe kanalen zoals op social media. Het overtreden van deze voorwaarden kan leiden tot een tijdelijke of permanente ban.
+
+### 3. Tijdelijk verbod
+
+**Impact op de gemeenschap**: een ernstige schending van gemeenschapsnormen, inclusief aanhoudend ongepast gedrag.
+
+**Gevolg**: een tijdelijk verbod op elke vorm van interactie of openbare
+communicatie met de gemeenschap gedurende een bepaalde periode. Geen publieke of privé-interactie met de betrokken personen, inclusief ongevraagde interactie met degenen die de Gedragscode handhaven, is toegestaan ​​tijdens deze periode.
+Het overtreden van deze voorwaarden kan leiden tot een permanente ban.
+
+### 4. Permanente ban
+
+**Impact op de gemeenschap**: een patroon van schending van de gemeenschapsstandaarden vertonen, waaronder aanhoudend ongepast gedrag, intimidatie van een individu, of agressie jegens of kleinering van groepen van individuen.
+
+**Gevolg**: een permanent verbod op elke vorm van openbare interactie in de gemeenschap.
```

Comment:
I'm getting confused about the public and private interactions 😅. Here it only describes public interactions with the community (as it does in the EN translation). It does not mention public/private interactions with people involved. Is this intentional?

Reply:
I'd assume it'd be public & private given the severity - but a great discussion for the EN one indeed